src/core/evaluator.js

```diff
@@ -377,6 +381,34 @@ class PartialEvaluator {
     return data;
   }
 
+  async fetchStandardFontData(name) {
+    const standardFontNameToFileName = getStdFontNameToFileMap();
+    const filename = standardFontNameToFileName[name];
+    if (this.options.standardFontDataWorkerFetch) {
+      const url = `${this.options.standardFontDataUrl}${filename}.pfb`;
+      const response = await fetch(url);
+      if (response.ok) {
+        const data = new Uint8Array(await response.arrayBuffer());
+        return new Stream(data);
```

Comment:
Please note that the `Stream` constructor will wrap an `ArrayBuffer` into a `Uint8Array`: https://github.com/mozilla/pdf.js/blob/3da9f077be33d8689129d5ddb7915d7bf1f2a5fa/src/core/stream.js#L23-L26
Hence this can be changed to simply:
```suggestion
        return new Stream(await response.arrayBuffer());
```